easybuild/easyblocks/generic/scons.py

```diff
@@ -54,7 +54,8 @@ def test_step(self):
         Test with SCons 
         """
         cmd = "scons check"
-        (out, _) = run_cmd(cmd, log_all=True)
+        if self.cfg['runtest']:
+            (out, _) = run_cmd(cmd, log_all=True)
```

Comment:
I think `scons check` is different from `make test`, where `test` is a common target but it can be different, while I expect it's always `scons check`?